### DISCUSSION TITLE
Google Timeline export

### DISCUSSION THREAD

**Discussion by SvbZ3r0** [Freika/dawarich#111] [Q&A]
Has anybody managed to get the new Google on-device timeline export working? Mine always says "Export failed" and I end up with about 50 points of data from 2013. Since Google Takeout for location data doesn't work anymore, this is the only way to get this data.
I know this has nothing to do with the Dawarich itself, but I figured I have more chances of running into somebody who has done it successfully here.

There's a lot of similar questions on the Google Support forums and elsewhere, but no answer.
See:
https://support.google.com/maps/thread/281853706/location-history-timeline-export-fails?hl=en
https://support.google.com/maps/thread/256402406/google-s-new-timeline-won-t-let-me-export-my-data?hl=en
https://www.reddit.com/r/GoogleMaps/comments/1dx4hfc/google_maps_timeline_data_storage_on_android/?rdt=33369

![image](https://github.com/Freika/dawarich/assets/57280279/dc684fb7-7231-4aeb-872b-4e69ec558ea3)

**Comment by Nihisil** [comment]
Did you manage to find a solution? It was surprise to see 44kb file for my location history export today

**Reply by SvbZ3r0** [reply]
Nope. I will update here if I do find a fix. 
The infuriating thing is that they have removed the ability to get the data through any other means too; meaning, until there's a fix for this, we are straight out of luck.

**Reply by SvbZ3r0** [reply]
https://www.reddit.com/r/GoogleMaps/comments/1cwka7v/comment/ldzbcba/

Haven't verified yet, but if its true the data is gone forever. :(

**Reply by Freika** [reply]
> https://www.reddit.com/r/GoogleMaps/comments/1cwka7v/comment/ldzbcba/
> 
> Haven't verified yet, but if its true the data is gone forever. :(

this is insane

**Reply by SvbZ3r0** [reply]
Apparently the database itself isn't corrupted.
https://www.reddit.com/r/GoogleMaps/comments/1cwka7v/comment/lgwab8m/

**Comment by kienemaus** [comment]
I have got the device to provide the JSON - But i have no idea how to use it. The data format is completely different than the timeline takeout.

The device setting>location services>timeline>export timeline data works. Then i can upload the file to my google drive.

**Reply by SvbZ3r0** [reply]
It works. Unless it doesn't 🤷
The issue doesn't affect everybody. If it did, it would have been patched by now. 

As for the format difference, I think there was an update earlier that added compatibility for the new format?
